client/src/components/Notification/NotificationsMenuItem.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { Box, Divider, ListItemText, Menu, MenuItem as DropdownMenuItem, Badge } from '@mui/material';
+import { NavLink } from 'react-router-dom';
+import { Notification } from '../../interface/Notification';
+import Avatar from '@mui/material/Avatar';
+import { useStyles } from './useStyles';
+import {
+  getWindowDimensions,
+  getResource,
+  unreadNotifications,
+  formatMMDDYYYY,
+  getDescriptiveType,
+} from '../../helpers/NoficationsMenuItemHelper';
+
+const NotificationsMenuItem = (notifications: [Notification], readNotifications: () => void) => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const classes = useStyles();
+  const handleMenuOpen = (event: React.MouseEvent<HTMLElement>) => {
+    readNotifications();
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = (notification?: Notification) => {
```

Comment:
I don't understand why we're taking a notification as a parameter here?

Reply:
i'm not sure either, removed.